frontend/src/main/pages/CourseDetails/CourseDetailsPage.js

```diff
@@ -1,12 +1,34 @@
 import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+import { useParams } from "react-router-dom";
+import { useBackend, _useBackendMutation } from "main/utils/useBackend";
+import CourseDetailsTable from "main/components/CourseDetails/CourseDetailsTable";
 
 export default function CourseDetailsPage() {
-  // Stryker disable all : placeholder for future implementation
+  let { yyyyq, enrollCd } = useParams();
+
+  const {
+    data: courseDetail,
+    _error,
+    _status,
+  } = useBackend(
+    // Stryker disable all : hard to test for query caching
```

Comment:
This should be matched with a `// Stryker restore all` as soon as the part you are not testing for is done.   Where is the matching line?

Reply:
Or change it to the `next-line` version

Reply:
Changed to next-line version!